### PR TITLE
DynamicForm - disable save button

### PR DESCRIPTION
**Pull request by srpmtt**
| Q               | A
| --------------- | ---
| Bug fix?        | [ ]
| New feature?    | [X]
| New sample?      | [ ]
| Related issues?  | mentioned in #1803

DynamicForm _saveDisabled_ prop.
Handle enabling/disabling form save button.

### CODE REVIEW

File: src/controls/dynamicForm/IDynamicFormProps.ts [hunk 1]
```diff
@@ -13,6 +13,10 @@ export interface IDynamicFormProps {
    * Specifies if the form is disabled
    */
   disabled?: boolean;
+  /**
+   * Specify if save button is disabled.
+   */
+  saveDisabled?: boolean;
```
Comment (michaelmaillot) [inline src/controls/dynamicForm/IDynamicFormProps.ts:19]: Please update the control's doc page to reflect the new prop added.

**Review by michaelmaillot — CHANGES_REQUESTED** [review]
Hi @srpmtt,

Thanks for this PR! A little update requested and we'll be good!

File: src/controls/dynamicForm/IDynamicFormProps.ts (hunk 1 above)
Comment (srpmtt) [reply src/controls/dynamicForm/IDynamicFormProps.ts:19]: updated DynamicForm.md with the new prop

**Comment by michaelmaillot** [comment]
Merged manually, thank you!

**Pull request merged**